### PR TITLE
Add ruleset for Blogtrottr.com

### PR DESCRIPTION
**Pull request by Injust**

### CODE REVIEW

File: src/chrome/content/rules/Blogtrottr.com.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!--
+	Connection closed:
+	- blog.blogtrottr.com
+-->
+
+<ruleset name="Blogtrottr.com">
+	<target host="blogtrottr.com" />
+	<target host="www.blogtrottr.com" />
+	<target host="static.blogtrottr.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Blogtrottr.com.xml]: Please also add `support.blogtrottr.com`

Comment (Injust) [reply src/chrome/content/rules/Blogtrottr.com.xml]: Added, thank you.

**Comment by Injust** [comment]
@J0WI How do you usually find these subdomains? I try to parse the page source, check Google, and go through the SANs in the SSL cert, but always seem to have a few subdomains slip by.

**Comment by J0WI** [comment]
See https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#enumerating-subdomains

**Pull request merged**